app/components/account/danger-zone.js

```diff
@@ -33,10 +33,14 @@ export default Component.extend({
         .then(() => {
           this.authManager.logout();
           this.routing.transitionTo('index');
-          this.notify.success(this.l10n.t('Your account has been deleted successfully.'));
+          this.notify.success(this.l10n.t('Your account has been deleted successfully.'), {
+            id: 'Account_Delete'
```

Comment:
make the first letter small for all ids.

Reply:
Sure

Reply:
@CosmicCoder96 Please Review